cfme/common/__init__.py

```diff
@@ -365,6 +367,64 @@ def step(self, *args, **kwargs):
         self.prerequisite_view.toolbar.policy.item_select('Manage Policies')
 
 
+class CompareView(BaseLoggedInPage):
+    """generic class for compare views"""
+
+    @View.nested
+    class toolbar(View):
+        all_attributes = Button(title="All attributes")
+        different_values_attributes = Button(title="Attributes with different values")
+        same_values_attributes = Button(title="Attributes with same values")
+        details_mode = Button(title="Details Mode")
+        exists_mode = Button(title="Exists Mode")
+        # add expanded view, compressed view buttons.
+        download = Dropdown('Download')
+        view_selector = View.nested(ItemsToolBarViewSelector)
+
+
+class ComparableMixin:
+    """
+    Mixin for comparing entities. Should be added to Collection class.
+    Constants:
+            DROPDOWN_TEXT: (str) the text in the Configuration dropdown for comparing entities
+            NAV_STRING: (str) used as second arg in call to navigate_to()
+            COMPARE_VIEW_PROVIDER: (VIEW) view class to create and return when filtering on provider
+            COMPARE_APP_VIEW: (VIEW) view class to create and return when no provider filtering
```

Comment:
To accomodate the name change
```suggestion
            COMPARE_VIEW_ALL: (VIEW) view class to create and return when no provider filtering
```